File: Models/Models/ViewModels/CategoryViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CorporateQnAModels.Models.ViewModels
{
    public class CategoryViewModel
    {
		public int CategoryId { get; set; }
		public string CategoryName { get; set; }
		public string CategoryDescription { get; set; }
		public int TotalTags { get; set; }
		public int WeekTags { get; set; }
		public int MonthTags { get; set; }
		
	}
}

File: CorporateQnA/CorporateQnAApp/app/models/categoryViewModel.ts
export class CategoryViewModel{
     categoryId:number;
     categoryName:string;
     categoryDescription:string;
     totalTags:number;
     weekTags:number;
     monthTags:number;

     constructor(args:any){
          this.categoryId=args.categoryId;
          this.categoryName=args.categoryName;
          this.categoryDescription=args.categoryDescription;
          this.totalTags=args.totalTags;
          this.weekTags=args.weekTags;
          this.monthTags=args.monthTags;
     }
}